sdk/table/azure-table/azure/table/_table_client.py

```diff
@@ -0,0 +1,509 @@
+        :type row_key: str
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: None
+        :rtype: None
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        if_match, if_not_match = _get_match_headers(kwargs=dict(kwargs, etag=etag, match_condition=match_condition),
+                                                    etag_param='etag', match_param='match_condition')
+
+        try:
+            self._client.table.delete_entity(
+                table=self.table_name,
+                partition_key=partition_key,
+                row_key=row_key,
+                if_match=if_match or if_not_match or '*',
+                **kwargs)
+        except ResourceNotFoundError:
+            raise ResourceNotFoundError
+
+    @distributed_trace
+    def create_entity(
+            self,
+            table_entity_properties,  # type: dict
+            query_options=None,  # type: Optional[QueryOptions]
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> Entity
+        """Insert entity in a table.
+
+        :param table_entity_properties: The properties for the table entity.
+        :type table_entity_properties: dict[str, object]
+        :param query_options: Parameter group.
+        :type query_options: ~azure.table.models.QueryOptions
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: Entity mapping str to object
+        :rtype: Entity
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        if table_entity_properties:
+
+            if "PartitionKey" in table_entity_properties and "RowKey" in table_entity_properties:
+                table_entity_properties = _add_entity_properties(table_entity_properties)
+            else:
+                raise ValueError
+        try:
+
+            inserted_entity = self._client.table.insert_entity(
+                table=self.table_name,
+                table_entity_properties=table_entity_properties,
+                query_options=query_options,
+                **kwargs
+            )
+            properties = _convert_to_entity(inserted_entity)
+            return Entity(properties)
+        except ValueError as error:
+            process_storage_error(error)
+
+    @distributed_trace
+    def update_entity(  # pylint:disable=R1710
+            self,
+            mode,  # type: Any
+            table_entity_properties,  # type: dict
+            partition_key=None,  # type: Optional[str]
+            row_key=None,  # type: Optional[str]
+            etag=None,   # type: Optional[object]
+            match_condition=None,  # type: Optional[MatchCondition]
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        """Update entity in a table.
+
+        :param mode: Merge or Replace entity
+        :type mode: ~azure.table._models.UpdateMode
```

Comment:
if UpdateMode is defined in __init__.py then you can drop the _models. in the reference